### ISSUE TITLE
Design: Explore ways to expose permissions within the Members table

### ISSUE THREAD

**Issue by Katherine-Osos** [cisagov/manage.get.gov#2668]
### Issue description

For org model, admins need the ability to quickly scan the permissions assigned to each member of their organization. This information would ideally be exposed in the Members table, which already provides other info about members. 

A few ideas have already been discussed, but we need to continue refining and determine the best path forward for exposing this information.

### Acceptance criteria

- [ ] Create design for exposing member permissions in the Members table
- [ ] Create dev ticket for implementation

### Additional context

Ideas discussed so far: 
- Column with individual chips for each permission, stacked. Permissions are worded as roles.
  - domain requester, domain manager
- Column with separate chips for elements that a user can view vs elements a user can manage.
  - View: requests, members
  - Manage: domains
- Separate columns for elements a user can view vs elements a user can manage

**Comment by witha-c** [comment]
@gabydisarli take a look at the concept [here](https://miro.com/app/board/uXjVLRwseMY=/?moveToWidget=3458764606640077412&cot=14). This specific convo is still evolving, but related!

**Comment by PaulKuykendall** [comment]
In Sprint planning, This may not be required for initial launch. Consider this after user feedback.

**Comment by katypies** [comment]
Agreed with @PaulKuykendall - once we decide on the path for adding members with permissions, this table should reflect the minimum needed to convey the permissions (we can iterate and add more details in later work).

**Comment by h-m-f-t** [comment]
We're gonna call this blocked by the filters work. We need to learn a bit more about what users need w/r/t permissions and don't want to delve into this until we're at least deeper into offering a filtered view.

**Comment by gabo0oo** [comment]
We might want to prioritize working on this before we implement the new invitation experience.
